pkg/tagger/collectors/docker_extract_test.go

```diff
@@ -101,6 +101,28 @@ func TestDockerRecordsFromInspect(t *testing.T) {
 			},
 			expectedHigh: []string{"mesos_task:system_dd-agent.dcc75b42-4b87-11e7-9a62-70b3d5800001"},
 		},
+		{
+			testName: "extractAuroraScheduler",
```

Comment:
You might want to add a negative case: i.e. where one of the tags is missing...to show that you successfully get an empty map in that case.

Reply:
Thanks @etotten!  Added additional `extractAuroraSchedulerNonAuroraJob` test and updated existing `testName` to `extractAuroraSchedulerAuroraJob`.